Lib/shutil.py

```diff
@@ -1249,11 +1249,17 @@ def disk_usage(path):
         return _ntuple_diskusage(total, used, free)
 
 
-def chown(path, user=None, group=None):
+def chown(path, user=None, group=None, dir_fd=None, follow_symlinks=True):
```

Comment:
These should be keyword-only arguments, like in `os.chown`:

```suggestion
def chown(path, user=None, group=None, *, dir_fd=None, follow_symlinks=True):
```